src/common.h

```diff
@@ -100,14 +100,18 @@ typedef Bytes48 KZGProof;
 
 /** Stores the setup and parameters needed for computing KZG proofs. */
 typedef struct {
-    /** The length of `roots_of_unity`, a power of 2. */
-    uint64_t max_width;
-    /** Powers of the primitive root of unity determined by `SCALE2_ROOT_OF_UNITY` in bit-reversal
-     * permutation order, length `max_width`. */
-    fr_t *roots_of_unity;
-    /** The expanded roots of unity. */
+    /** The size of our multiplicative subgroup (the roots of unity). This is the size of
+     *  the extended domain (after the RS encoding has been applied), so the size of
+     *  the subgroup is FIELD_ELEMENTS_PER_EXT_BLOB. */
+    uint64_t domain_size;
+    /** Roots of unity in bit-reversal permutation order.
```

Comment:
Comments here are much better than before!
Nit: "bit-reversal permutation order" vs. "bit-reversed order" vs "reverse bit order(ing)" (in some blog posts). 
We should have a consistent name for that, at least within this code.

Reply:
Took a stab at it in cb63fb0d7089a2c1076af78c16a12e93ce877ddf
Feel free to suggest alternatives.